app/services/foreman/plugin.rb

```diff
@@ -351,6 +351,11 @@ def parameter_filters(klass)
       @parameter_filters.fetch(klass.is_a?(Class) ? klass.name : klass, [])
     end
 
+    # Add concerns to a host
+    def extend_host(concern_module)
```

Comment:
I don't say plugin devs should not know about internal objects, I don't want them to use them directly. Otherwise plugin gets broken without us knowing, leading to plugin devs frustration etc. I find this very important, plugins need more stable API internal that they can use.